libs/sdk-backend-tiger/src/backend/workspace/attributes/index.ts

```diff
@@ -61,6 +71,26 @@ export class TigerWorkspaceAttributes implements IWorkspaceAttributesService {
         });
     }
 
+    public getCatalogAttributes(refs: ObjRef[]): Promise<ICatalogAttribute[]> {
+        const filter = refs
+            .map(async (ref) => {
+                const id = await objRefToIdentifier(ref, this.authCall);
+                return `attributes.id=${id}`;
+            })
+            .join(",");
+
+        return this.authCall(async (client) => {
+            const catalogItems = await loadAttributesAndDateDatasets(
+                client,
+                this.workspace,
+                filter.length < MAX_FILTER_LENGTH ? filter : "",
```

Comment:
pls move this line into constant like:

const rsqlFilter = filter.length < MAX_FILTER_LENGTH ? filter : "";

and add coment to this line something like: when filers are too long then we fetch all of them and use this separate const in loadAttributesAndDateDatasets request.